webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -34,13 +34,13 @@
       <v-btn
         v-show="isMobile && mobileFilter || !isMobile"
         class="btn px-2 btn-primary"
+        :min-width="iconWidth"
         outlined
         @click="openPeopleAdvancedFilterDrawer()">
-        <v-icon small class="primary--text mr-1">fa-sliders-h</v-icon>
-        <span class="d-none font-weight-regular caption d-sm-inline mr-1">
-          {{ $t('profile.label.search.openSearch') }}
-          {{ advancedFilterCountDisplay }}
-        </span>
+        <v-icon small class="primary--text mr-lg-1">fa-sliders-h</v-icon>
```

Comment:
```suggestion
        <v-icon small class="primary--text me-lg-1">fa-sliders-h</v-icon>
```
RTL, remember we don't use `mr` nor `ml` anymore.
Besides, why adding margin to an icon, by experience, it's not a good behavior.

Reply:
done!